packages/multi-column/src/pm-nodes/Column.ts

```diff
@@ -20,6 +22,45 @@ export const Column = createStronglyTypedTiptapNode({
   priority: 40,
   // defining: true, // TODO
 
+  addAttributes() {
+    return {
+      width: {
+        // Why does each column have a default width of 1, i.e. 100%? Because
```

Comment:
Clear! Let's think whether we want to use "1" or "100%" as attribute

could we also use `flex-grow`? or is `flex-basis` better because it takes px values?